### PR TITLE
Simplified is removed from class names in task 3

### PR DESCRIPTION
**Pull request by EBGlom**
Please fix typos as you find them and push to this branch. We'll merge before the job ad is published

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,23 +30,23 @@ Be aware that the ```MooringChain``` class and its methods expect and return SI
...
 
-The task 3 Python file contains a  ```SimplifiedFloater``` class, and ```SimplifiedBarge``` and ```SimplifiedCylinder``` sub classes that can return added mass and natural period for a cylinder and a barge in accordance with approximated methods
-This class can be used to answer the following questions:
- - a) What is the added mass of a cylinder with diameter = 100m?
- - b) What is the natural period of a barge with mass = 150,000t, length=300m, width=55m and draft=18m? Hint, first calculate the estimated added mass through the estimated_added_mass method, then the heave natural period
+The task 3 Python file contains a  ```Floater``` class, and ```Barge``` and ```Cylinder``` sub-classes that return added mass and natural period for a cylinder and a barge in accordance with approximated methods.
+The classes can be used to answer the following questions:
+ - What is the added mass of a cylinder with diameter = 100m? (Give the answer in tons)
+ - What is the natural period of a barge with mass = 150,000t, length=300m, width=55m and draft=18m? Hint, first calculate the estimated added mass through the estimated_added_mass method, then the heave natural period
```
Comment (EBGlom) [inline README.md]: Should we remove this hint? This is done automatically, now

Comment (EBGlom) [reply README.md]: Removed hint

File: README.md [hunk 2]
```diff
@@ -30,25 +30,25 @@ Be aware that the ```MooringChain``` class and its methods expect and return SI
 ### 3) Added mass and natural period
 Global performance and natural period is important parameters for a floating structure.
 
-The task 3 Python file contains a  ```SimplifiedFloater``` class, and ```SimplifiedBarge``` and ```SimplifiedCylinder``` sub classes that can return added mass and natural period for a cylinder and a barge in accordance with approximated methods
-This class can be used to answer the following questions:
- - a) What is the added mass of a cylinder with diameter = 100m?
- - b) What is the natural period of a barge with mass = 150,000t, length=300m, width=55m and draft=18m? Hint, first calculate the estimated added mass through the estimated_added_mass method, then the heave natural period
+The task 3 Python file contains a  ```Floater```, ```Barge``` and ```Cylinder``` classes that return added mass and natural period for a cylinder and a barge in accordance with approximated methods.
```
Comment (larsmr) [inline README.md]: Consider "The task 3 Python file contains ```Floater```, ```Barge``` and ```Cylinder```  classes ..." or "The task 3 Python file contains a ```Floater```,  a ```Barge``` and a ```Cylinder```  class "?

Comment (EBGlom) [reply README.md]: Updated as proposed

**Pull request merged**